README.md

```diff
@@ -1 +1,33 @@
-# list-view-controls
\ No newline at end of file
+[![Build Status](https://travis-ci.org/mendixlabs/list-view-controls.svg?branch=master)](https://travis-ci.org/mendixlabs/list-view-controls)
+[![Dependency Status](https://david-dm.org/mendixlabs/list-view-controls.svg)](https://david-dm.org/mendixlabs/list-view-controls)
+[![Dev Dependency Status](https://david-dm.org/mendixlabs/list-view-controls.svg#info=devDependencies)](https://david-dm.org/mendixlabs/list-view-controls#info=devDependencies)
+[![codecov](https://codecov.io/gh/mendixlabs/list-view-controls/branch/master/graph/badge.svg)](https://codecov.io/gh/mendixlabs/list-view-controls)
+
+# List view controls
+Enable users to filter or search a list view.
+
+## Available widgets
+* `Check box filter`
+* `Drop down filter`
+* `Drop down sort`
+* `Text box search`
```

Comment:
Remind that this README is going to be the app store readme page, so make a short 1 line description what the widget does. Move the other readme files into folder `docs` and dont start there name with REAMDE

Add a single screen shot with all different widgets in there, as Danny if he has one.

Reply:
Done